app/models/intake.rb

```diff
@@ -607,4 +603,8 @@ def update_or_create_14446_document(filename)
     )
     document
   end
+
```

Comment:
I'm guessing this is what gets used in the views to display the delay message? Based on the standup convo yesterday, it seems like this works out of the box for us, with no updates needed, right?